test/WebJobs.Script.Tests/Configuration/LanguageWorkerOptionsSetupTests.cs

```diff
@@ -31,6 +31,26 @@ public void LanguageWorkerOptions_Expected_ListOfConfigs(string workerRuntime)
             {
                 testEnvironment.SetEnvironmentVariable(RpcWorkerConstants.FunctionWorkerRuntimeSettingName, workerRuntime);
             }
+            else
+            {
+                // dotnet-isolated worker config requires placehoolder mode env variable to be set to 1.
+                testEnvironment.SetEnvironmentVariable(EnvironmentSettingNames.AzureWebsitePlaceholderMode, "1");
```

Comment:
nit: Can we clarify this with something like:

`// The dotnet-isolated worker only runs in placehoolder mode. Setting the placeholder environment to 1 for the test.

Reply:
Fixed language to suggested.